ngraph/core/include/ngraph/dimension.hpp

```diff
@@ -61,6 +61,11 @@ namespace ngraph
         /// \brief Return the interval of valid lengths
         const Interval& get_interval() const { return m_dimension; }
         Interval& get_interval() { return m_dimension; }
+
+        /// \brief Return the dimension name
+        const std::string& get_name() const { return m_name; }
+        /// \brief Return the dimension name
+        bool has_constant_name() const { return is_static() && m_name.empty(); }
```

Comment:
consider deleting the method -- appeared that it is not going to be widely used.

Reply:
What was the initial idea of this method?